src/utils/utils.cpp

```diff
@@ -719,22 +719,21 @@ QString dumpCertificate(const QSslCertificate &cert)
     if (cert.isNull())
       return "\n-\n";
 
-    QString s;
-    QString s_none = QObject::tr("<Not Part of Certificate>");
+    QString s = "\n";
 #if (QT_VERSION >= QT_VERSION_CHECK(5, 0, 0))
-    #define CERTIFICATE_STR(x) ( ((x).join("") == "" ) ? s_none : (x).join(";") )
+    s += cert.toText();
 #else
+    QString s_none = QObject::tr("<Not Part of Certificate>");
     #define CERTIFICATE_STR(x) ( ((x) == "" ) ? s_none : (x) )
-#endif
-
 
-    s += "\nIssued To\n";
+    s += "Certificate:\n";
+    s += "\nIssued To:\n";
     s += "CommonName(CN):             " + CERTIFICATE_STR(cert.subjectInfo(QSslCertificate::CommonName)) + "\n";
     s += "Organization(O):            " + CERTIFICATE_STR(cert.subjectInfo(QSslCertificate::Organization)) + "\n";
     s += "OrganizationalUnitName(OU): " + CERTIFICATE_STR(cert.subjectInfo(QSslCertificate::OrganizationalUnitName)) + "\n";
     s += "Serial Number:              " + dumpHexPresentation(cert.serialNumber()) + "\n";
 
-    s += "\nIssued By\n";
+    s += "\nIssued By:\n";
     s += "CommonName(CN):             " + CERTIFICATE_STR(cert.issuerInfo(QSslCertificate::CommonName)) + "\n";
     s += "Organization(O):            " + CERTIFICATE_STR(cert.issuerInfo(QSslCertificate::Organization)) + "\n";
     s += "OrganizationalUnitName(OU): " + CERTIFICATE_STR(cert.issuerInfo(QSslCertificate::OrganizationalUnitName)) + "\n";
```

Comment:
Another way to do it is to define `s` as a `QStringList`, and finally use `s.join("\n")` to concatenate all the lines.